### PR TITLE
Fix #3: add-CODEOWNERS template

### LINKED ISSUE #3: Codeowner template
Add Codeowner template
Mention @rt4914 as codeowner for now.

References:
https://github.com/forem/forem/blob/main/.github/CODEOWNERS
https://github.com/oppia/oppia-android/blob/develop/.github/CODEOWNERS

### PR DESCRIPTION
**Pull request by KvngJamesNG**
Fix #3: add-CODEOWNERS template

### CODE REVIEW

**Comment by rt4914** [comment]
@KvngJamesNG Any idea when can you update this PR? This PR is blocking other developers as it is related to codeowners.

**Comment by KvngJamesNG** [comment]
@rt4914 what else should I do ?

**Comment by rt4914** [comment]
@KvngJamesNG I cannot review it until you assign it to me. Have a look at point 6 https://github.com/rt4914/KlinicCon-Backend/wiki#creating-a-pull-request

Also have a look at this video too: https://www.loom.com/share/8fc0c72c462a4eff9713f28dd82898e6?sid=d437dec2-4180-4d33-aaeb-bf43cc5f507b
Currently this PR is in no state to review because of:
1. Not assigned to.
2. Draft PR.
3. PR Title is bad
4. PR does not have a any description
5. Not linked to issue.

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,52 @@
+                                                              # Codeowners for the project
+
+
+
```
Comment (rt4914) [inline CODEOWNERS:4]: Remove empty spaces above line 5th.

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# Global owner for everything in this repo
+* @rt4914
+
+# Owners/teams for backend (Ruby on Rails)
```
Comment (rt4914) [inline CODEOWNERS:8]: Remove everything below this.

**Review by rt4914 — CHANGES_REQUESTED** [review]
As per my previous comment make changes to PR Title, description.
1. Have a look at https://github.com/rt4914/KlinicCon-Backend/wiki
2. Also next time please make sure PR is in `Non-Draft` state before assigning for review.

File: CODEOWNERS (hunk 1 above)
Comment (KvngJamesNG) [reply CODEOWNERS]: I have done this...

File: CODEOWNERS (hunk 2 above)
Comment (KvngJamesNG) [reply CODEOWNERS:5]: I have also done this and assigned it to you as well

Comment (rt4914) [reply CODEOWNERS:5]: @KvngJamesNG You can get rid of last line too. We don't need that comment anymore.

**Review by rt4914 — CHANGES_REQUESTED** [review]
The linking of issue is incorrect.

#15 is the PR number (this creates a cyclic linking)
Issue number is #3

So use `#3` in title and description.

Also in future make sure you follow these steps to create a correct branch name.
https://github.com/rt4914/KlinicCon-Backend/wiki#how-to-contribute

Example for this PR the branch name should have been `KvngJamesNG/add-codeowners` or something similar. 

This you can keep in mind for your next PRs.

**Review by rt4914 — APPROVED** [review]
LGTM (Looks good to me)

Congratulations on your first PR 🚀

**Comment by KvngJamesNG** [comment]
Thank you a lot @rt4914

**Pull request merged**